Cargo.lock

```diff
@@ -1394,32 +1382,32 @@ checksum = "712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
 
 [[package]]
 name = "windows"
-version = "0.58.0"
+version = "0.59.0"
```

Comment:
While github.com/mozilla-firefox/firefox is on `windows` `0.58.0`, I would prefer Neqo to stay on `windows` `0.58.0`.

https://searchfox.org/mozilla-central/rev/2fc84c3d7aba20db92741279f0b4ac9173635b5c/Cargo.lock#7767-7774

Reply:
Just force-pushed to the parent of the tip, to exclude the `chore: extend `windows` dep. to include 0.59` commit.